src/main/webapp/resources/js/pages/analysis/components/Analysis.jsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * This file is responsible for displaying the
+ * tabs required depending on the analysis state
+ * and analysis type.
+ */
+
+/*
+ * The following import statements makes available all the elements
+ *required by the components encompassed within
+ */
+
+import React, { useContext } from "react";
+import { Tabs, Typography, Icon } from "antd";
+
+import { AnalysisDetails } from "./AnalysisDetails";
+import { AnalysisSteps } from "./AnalysisSteps";
+import { AnalysisProvenance } from "./AnalysisProvenance";
+import { AnalysisOutputFiles } from "./AnalysisOutputFiles";
+import { AnalysisPhylogeneticTree } from "./AnalysisPhylogeneticTree";
+import { AnalysisBioHansel } from "./AnalysisBioHansel";
+import { AnalysisSistr } from "./AnalysisSistr";
+import { AnalysisError } from "./AnalysisError";
+import { getI18N } from "../../../utilities/i18n-utilties";
+import { SPACE_MD } from "../../../styles/spacing";
+
+import {
+  AnalysisContext,
+  AnalysisProvider
+} from "../../../contexts/AnalysisContext";
+import { AnalysisDetailsProvider } from "../../../contexts/AnalysisDetailsContext";
+
+const TabPane = Tabs.TabPane;
+const { Title } = Typography;
+
+// Built in Analysis Types
+const analysisTypesWithAdditionalPage = [
+  "BIO_HANSEL",
+  "SISTR_TYPING",
+  "PHYLOGENOMICS",
+  "MLST_MENTALIST"
+];
+
+export default function Analysis() {
+  const { analysisContext, analysisContextUpdateSubmissionName } = useContext(
+    AnalysisContext
+  );
+
+  /*
+   * The following renders the tabs, and selects the
+   * tab depending on the state and type of analysis.
+   * The steps the analysis has gone through or is
+   * currently running through are only displayed
+   * if the analysis has not completed. If successfully
+   * completed then a green check mark is displayed next
+   * to the analysis name.
+   */
+  return (
+    <>
+      <div
```

Comment:
This `div` wraps the whole component so you do not need the `<></>` surrounding the entire thing.  Also when you have multiple attributes like this it is better to use `styled-components`.

Reply:
Fixed in 2804c23baa10fdf170ebd8b07a191142d0927b12